accessioning/digitalmediaseparation.md

```diff
@@ -105,6 +105,8 @@ An example of a completed record for separation:
 
 ![](/digitalmediaseparation/media/image1.png)
 
+Deprecated
+{: .label .label-red }
```

Comment:
Is it possible for this label to be in line with the section 5 header or below it? Just so it's closer to what is being deprecated.